modules/bibformat/lib/elements/bfe_record_recommendations.py

```diff
@@ -23,34 +23,31 @@
 * Based on Obelix recommendations and word similarity ranking
 """
 
+import re
+
 from invenio.config import CFG_OBELIX_HOST
 from invenio.messages import gettext_set_language
 
 
 html_script = """
-<div class="recommendations_content" style="display:none;"></div>
 <script>
 $( document ).ready(function() {
     $.getJSON("%(recommendations_url)s" , function( data ) {
-        var root = $( "div.recommendations_content" );
+        var root = $( "<div/>", {class: 'recommendations'
```

Comment:
you are missing a comma after class: 'recommendations', please add it and update the PR